addons/interaction/functions/fnc_laptop_addActionClose.sqf

```diff
@@ -0,0 +1,15 @@
+/**
+ * Opens a laptop
```

Comment:
Why is this laptop specific?

Reply:
Only the laptop has that specific open/close animation. The is already an abstraction layer in config, using the following asset neutral settings:

			openAction = 
			closeAction =